src/longopt/grdedit_inc.h

```diff
@@ -25,42 +25,28 @@ static struct GMT_KEYWORD_DICTIONARY module_kw[] = {
 		  short_directives,    long_directives,
 		  short_modifiers,     long_modifiers,
 		  transproc_mask */
-	{ 0, 'A', "",
-	          "",                  "",
-	          "",                  "",
-		  GMT_TP_STANDARD },
-	{ 0, 'C', "",
-	          "",                  "",
-	          "",                  "",
-		  GMT_TP_STANDARD },
-	{ 0, 'D', "",
-	          "",                  "",
+	{ 0, 'A', "adjust_inc",        "", "", "", "", GMT_TP_STANDARD },
+	{ 0, 'C', "cmdhist|command_history",
+	          "b,c,n,p",           "both,current,none,previous",
 	          "",                  "",
 		  GMT_TP_STANDARD },
-	{ 0, 'E', "",
-	          "",                  "",
+	{ 0, 'D', "netcdf|netCDF|ncheader",
 	          "",                  "",
+	          "c,d,n,o,r,s,t,v,x,y,z",
+		                       "cpt,dname,invalid,offset,remark,scale,title,varname,xname,yname,zname",
 		  GMT_TP_STANDARD },
-	{ 0, 'G', "",
-	          "",                  "",
+	{ 0, 'E', "transform",
+	          "a,e,h,l,r,t,v",     "hvflip,xyswap,hflip,rot90counterclock,rot90clock,transpose,vflip",
```

Comment:
the rot90... becomes very long.
Can we do (at least as alternative) "rotleft" and "rotright". It should be obvious that it is by 90º.

Reply:
******Maybe** rot90ccw and rot90cw are more specific than just a rotation of unspecific value.  I agree that **rot90counterclock** is too long and even not long enough since it begs for the "wise" at the end.

Reply:
Well, it would be hard to rotate a 2D grid by anything else than increments of 90º. But, yes, I would also prefer "rot90ccw" and "rot90cw"